src/components/CarDetails/CarDetails.jsx

```diff
@@ -90,7 +90,7 @@ const CarDetails = ({ car, onGoBack }) => {
               {' '}
               <box-icon name="undo" color="#fff" />
             </button>
-            <p className="text-white fw-semibold mb-1">Configure</p>
+            <p className="text-white fw-semibold mb-1">Reserve</p>
```

Comment:
_note: Since this PR does not include all the content, I'll include my comments here:_

- [ ] This is the test that comes with React, and it's not passing:

![image](https://github.com/flemton/Final-Capstone-Frontend/assets/57421823/10bf3460-cb58-47e7-b314-f2ffa4f7956a)

Kindly remove it if not using it. 👍 

- [ ] You have not included any frontend tests:

![image](https://github.com/flemton/Final-Capstone-Frontend/assets/57421823/68381b8b-f312-4771-bc98-0fa7b301c9d9)

Kindly either add some or remove this section on the readme. 👍 

- [ ] Here, it is perfectly possible to click the "Reserve" button without the required fields:

![image](https://github.com/flemton/Final-Capstone-Frontend/assets/57421823/7318251d-36c3-4bca-a9eb-fdc53bcbaf8f)

![image](https://github.com/flemton/Final-Capstone-Frontend/assets/57421823/38d096c5-1cfb-45dd-aa04-ea23bdd3e8b5)

Kindly, add some validation to ensure all the required fields have been filled by the user. 👍 

- [ ] OPTIONAL: There is no way to change the user (logout):

![image](https://github.com/flemton/Final-Capstone-Frontend/assets/57421823/43d64c48-a7d6-4f46-81cc-949aad603edf)

The only way to logout is deleting the local storage.

Kindly, add a way to logout.  It could be a simple button or link that logsout and the redirects to the login page.

_tip:  In order to logout, just remove the local storage and redirect to login._
_tip 2:  This shouldn't take more than 10 minutes, so don't worry._